appveyor.yml

```diff
@@ -14,7 +14,7 @@ cache:
 
 branches:
   only:
-    - master
+    - development
```

Comment:
Are you able to update `.travis.yml` while we're in here?

https://github.com/desktop/desktop/blob/249c06d51332a7254de848e1c77a60e1ae52d587/.travis.yml#L25

Reply:
yes! thanks for catching that!